app/config/runtimes.php

```diff
@@ -6,7 +6,7 @@
 /**
  * List of Appwrite Cloud Functions supported runtimes
  */
-$runtimes = new Runtimes();
+$runtimes = new Runtimes(App::getEnv('_APP_EXECUTOR_VERSION'));
```

Comment:
Can we have a default here?

```suggestion
$runtimes = new Runtimes(App::getEnv('_APP_EXECUTOR_VERSION', 'v2'));
```